password_security/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,105 @@
+            </xpath>
+
+            <!-- Move the settings box to desired location -->
+            <xpath expr="//div[@id='enable_password_reset']" position="after">
+                <xpath expr="//div[@id='password_policy']" position="move" />
+            </xpath>
+
+            <!-- Ensure our settings will come after any former customization -->
+            <xpath
+                expr="//div[@id='password_policy']//div[hasclass('o_setting_right_pane')]/*"
+                position="after"
+            >
+                <label string="Password Policy" for="password_expiration" />
+                <div class="content-group">
+                    <div class="mt16">
+                        <span>
+                            Password expires in
+                            <field name="password_expiration" class="oe_inline" />
+                            days.
+                        </span>
+                    </div>
+                    <div class="mt16">
+                        <span>
+                            User can change password in
+                            <field name="password_minimum" class="oe_inline" />
+                            hours again.
+                        </span>
+                    </div>
+                    <div class="mt16">
+                        <span>
+                            Disallow reuse of
+                            <field name="password_history" class="oe_inline" />
+                            previous passwords.
+                        </span>
+                        <div class="text-muted">
+                            Use negative number for infinite, or 0 to disable
+                        </div>
+                    </div>
+                    <div class="mt16">
+                        <span>
+                            Minimum number of lowercase characters
+                            <field name="password_lower" class="oe_inline" />
+                        </span>
+                    </div>
+                    <div class="mt16">
+                        <span>
+                            Minimum number of uppercase characters
+                            <field name="password_upper" class="oe_inline" />
+                        </span>
+                    </div>
+                    <div class="mt16">
+                        <span>
+                            Minimum number of numeric characters
+                            <field name="password_numeric" class="oe_inline" />
+                        </span>
+                    </div>
+                    <div class="mt16">
+                        <span>
+                            Minimum number of special characters
+                            <field name="password_special" class="oe_inline" />
+                        </span>
+                    </div>
+                </div>
+                <span id="minlength">
+                     Minimum number of characters
+                </span>
```

Comment:
I was doing some testing on my own instance and noticed this portion was slightly off
```suggestion
                      <div class="mt16">
                          <span id="minlength">
                               Minimum number of characters
                          </span>
                      </div>
                  </div>
```